src/Utility/TextFormatUtility.php

```diff
@@ -21,4 +23,18 @@ public static function formatNumber($unitPrice, $apiRoundingPrecision, $docPoint
     {
         return number_format($unitPrice, $apiRoundingPrecision, $docPoint, $thousandSep);
     }
+
+    /**
+     * Replace all accented chars by their equivalent non-accented chars.
+     *
+     * @param string $string
+     *
+     * @return string
+     */
+    public static function replaceAccentedChars(string $string): string
+    {
+        $transliterator = Transliterator::create('Any-Latin; Latin-ASCII');
```

Comment:
Maybe just in case check is this class exist and then proceed otherwise just return same string
```suggestion
        if (class_exists('Transliterator', false) && $transliterator = \Transliterator::create($string)) {
```